compiler/z/codegen/BinaryAnalyser.cpp

```diff
@@ -292,36 +292,26 @@ TR_S390BinaryAnalyser::genericAnalyser(TR::Node * root,
 void
 TR_S390BinaryAnalyser::longSubtractAnalyser(TR::Node * root)
    {
-   TR::Node * firstChild;
-   TR::Node * secondChild;
    TR::Instruction * cursor = NULL;
    TR::RegisterDependencyConditions * dependencies = NULL;
    bool setsOrReadsCC = NEED_CC(root) || (root->getOpCodeValue() == TR::lusubb);
    TR::InstOpCode::Mnemonic regToRegOpCode;
    TR::InstOpCode::Mnemonic memToRegOpCode;
    TR::Compilation *comp = TR::comp();
 
-   if (TR::Compiler->target.is64Bit() || cg()->use64BitRegsOn32Bit())
```

Comment:
You'll see a lot of these. On 64-bit `use64BitRegsOn32Bit` will always return false, and on 32-bit `use64BitRegsOn32Bit` will always return `true` so the query above will be true in all cases so we can fold away the resulting code.